src/cargo-kani/src/args_toml.rs

```diff
@@ -19,6 +20,16 @@ pub fn config_toml_to_args() -> Result<Vec<OsString>> {
     }
 }
 
+/// `cargo locate-project` produces a response like:
+/// `{"root":"/full/path/to/src/cargo-kani/Cargo.toml"}`
+fn cargo_locate_project() -> Result<PathBuf> {
+    let cmd = Command::new("cargo").arg("locate-project").output()?;
```

Comment:
One option is to run `cargo locate-project` with `--message-format plain` to avoid parsing json.
```
$ pwd
/home/ubuntu/git/kani/tests/cargo-kani/config/src
$ cargo locate-project
{"root":"/home/ubuntu/git/kani/tests/cargo-kani/config/Cargo.toml"}
$ cargo locate-project --message-format plain
/home/ubuntu/git/kani/tests/cargo-kani/config/Cargo.toml
```

Reply:
Done, though while debugging why everything suddenly failed (there is a newline in the bytes, oh), I realized I should update a bit of the error handling here too.

Reply:
Ouch!